src/components/UI/search-box/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { searchMemberContext } from '@store/search-members/searchMember-context';
+import classNames from './search-box.module.scss';
+
+const SearchBox = () => {
+  const { setSearchTerm } = searchMemberContext();
+  return (
+    <div className={classNames.searchContainer}>
+      <input
+        type="text"
+        placeholder="&#128269; search members here"
```

Comment:
What's this weird code? 🤔

Reply:
This is a search box for to search a name of member. I have uploaded a video above. You can watch it.